bin/maxwell-bootstrap

```diff
@@ -4,7 +4,7 @@ base_dir=$(dirname $0)/..
 
 CLASSPATH=$CLASSPATH:$base_dir/target/classes:../open-replicator/target/classes
 
-for file in $base_dir/target/maxwell-*/*/lib/*.jar lib/*.jar
+for file in $base_dir/target/dependency/*.jar lib/*.jar
```

Comment:
nope.  I'm done with Maven.  see the recent history of master, I've got a working `make` implementation of packaging.